drivers/aks/aks_driver.go

```diff
@@ -919,6 +922,11 @@ func (state state) hasLinuxProfile() bool {
 	return state.LinuxAdminUsername != "" && (state.LinuxSSHPublicKeyContents != "")
 }
 
+func (state state) hasHTTPApplicationRoutingSupport() bool {
```

Comment:
nit. Do we have a reference from Azure document for this? It would be nice to add the link here.

Reply:
I don't find any reference for this, it is only mentioned in the console UI and the API response:
![image](https://user-images.githubusercontent.com/5697937/61842506-124ad900-aecb-11e9-951c-6e2963785492.png)

```
$az aks enable-addons --resource-group gname --name cname --addons http_application_routing
Operation failed with status: 'Bad Request'. Details: Addon 'httpApplicationRouting' is not supported in this cloud environment.
```